packages/core/admin/admin/src/hooks/useFetchClient/index.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { useEffect } from 'react';
+import { getFetchClient } from '../../utils/getFetchClient';
+
+const cancelToken = () => {
+  const CancelToken = axios.CancelToken;
+  const source = CancelToken.source();
+
+  return source;
+};
+
+const useFetchClient = () => {
+  const source = cancelToken();
```

Comment:
i'd store this in a ref so the value never changes, this would also mean you can omit the eslint warning you're suppressing.

Reply:
```js
const source = useRef(null)

if(source.current === null) {
	source.current = cancelToken()
}

// ...
```